app/javascript/pages/admin/Reporting/User.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react'
+import {
+  Table,
+  Head,
+  HeaderRow,
+  HeaderCell,
+  Body,
+  Row,
+  Cell
+} from '@zendeskgarden/react-tables';
+import { Pagination } from '@zendeskgarden/react-pagination';
+import { useTranslation } from 'react-i18next'
+import * as R from 'ramda'
+
+const Reporting = ({ users }) => {
+  const pageSize = 10
+  const { t } = useTranslation()
+  const [ page, setPage ] = useState(1)
+  return (
+    <div>
+      <Table>
+        <Head>
+          <HeaderRow>
+            {/* TODO: searchable name */}
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.name')}</HeaderCell>
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.email')}</HeaderCell>
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.office')}</HeaderCell>
+            {/* TODO: toggle sort by hours */}
+            <HeaderCell>{t('volunteer_portal.admin.tab.reporting.hours')}</HeaderCell>
+          </HeaderRow>
+        </Head>
+        <Body>
+          {
+            R.pipe(
+              R.slice(pageSize * (page - 1), pageSize * page),
+              R.sort((a, b) => b.hours - a.hours),
+            )(users).map((user, index) =>
```

Comment:
I would move this out into a variable above the `return` statement. The benefit is separating data from display and decluttering the JSX part of the function.